roles/kube-init/tasks/main.yml

```diff
@@ -53,20 +53,28 @@
   # sudo chown $(id -u):$(id -g) $HOME/admin.conf
   # export KUBECONFIG=$HOME/admin.conf
 
+- name: Ensure .kube folder exists
+  file:
+    path: "{{ kubectl_home }}/.kube/"
+    state: directory
+    owner: "{{ kubectl_user }}"
+    group: "{{ kubectl_group }}"
+    mode: 0755
+
 - name: Copy admin.conf to kubectl user's home
   shell: >
-    cp -f /etc/kubernetes/admin.conf {{ kubectl_home }}/admin.conf
+    cp -f /etc/kubernetes/admin.conf {{ kubectl_home }}/.kube/admin.conf
```

Comment:
You can replace this with `synchronize` or `copy` with `remote_src: true`

Reply:
Also, you don't need a full `shell`, just `command` would also accomplish this.

Reply:
I'm going to leave this one here, but, I'll create an issue -- mostly because I don't wanna do a tear-down to retest just this portion.